Lecture 6/README.md

```diff
@@ -21,4 +21,4 @@ To learn more about the dataset, checkout its [registry page](http://registry.ml
 After this lecture, you are encouraged to work on two other image classification datasets available on Radiant MLHub. 
 1) [SEN12-FLOOD](http://registry.mlhub.earth/10.21227/w6xz-s898/): This dataset contains imagery of Sentinel-2 and Sentinel-1 satellites with labels for presence of flood in the imagery.
 
-2) [BigEarthNet](http://registry.mlhub.earth/10.14279/depositonce-10149/): This is a land cover classification dataset, and it contains ~570,000 images with multiclass labels for each image. This is a very large dataset, and if you are running it on a local computer, you may need to select a subset of the data for training. 
\ No newline at end of file
+2) [BigEarthNet](http://registry.mlhub.earth/10.14279/depositonce-10149/): This is a land cover classification dataset, and it contains ~570,000 images with multiclass labels for each image. This is a very large dataset, and if you are running it on a local computer, you may need to select a subset of the data for training. 
```

Comment:
@Daniel-Nwaeze Was there actually a change here? I can't see anything different between the two.